docs/products/kafka/howto/flink-with-aiven-for-kafka.rst

```diff
@@ -0,0 +1,200 @@
+Use Apache Flink® with Aiven for Apache Kafka®
+======================================================
+
+`Apache Flink® <https://flink.apache.org/>`_ is an open source platform for processing distributed streaming and batch data.
+Where Apache Kafka® excels at receiving and sending event streams, Flink consumes, transforms, aggregates, and enriches your data.
+
+The example in this article shows you how to create a simple Java Flink job that reads data from a Kafka topic, processes it, and then pushes it to a different Kafka topic.
+
+.. _kafka-flink-prereq:
+
+Prerequisites
+-------------
+
+You need an Aiven for Apache Kafka service up and running.
+
+Two topics named ``test-flink-input`` and ``test-flink-output`` should be created. Furthermore, for the example, you need to collect the following information about the Aiven for Apache Kafka service:
+
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service
+* ``APACHE_KAFKA_PORT``: The port of the Apache Kafka service
+
+You need to have Apache Maven™ installed to build the example.
+
+Setup the truststore and keystore
+''''''''''''''''''''''''''''''''''
+
+Create a :doc:`Java keystore and truststore <keystore-truststore>` for the Aiven for Apache Kafka service.
+For the following example we assume:
+
+* The keystore is available at ``KEYSTORE_PATH/client.keystore.p12``
+* The truststore is available at ``TRUSTSTORE_PATH/client.truststore.jks``
+* For simplicity, the same secret (password) is used for both the keystore and the truststore, and is shown here as ``KEY_TRUST_SECRET``
+
+
+Use Flink with Aiven for Apache Kafka
+-------------------------------------
+
+The following example shows how to customise the ``DataStreamJob`` generated from the `Quickstart <https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/configuration/overview/>`_ to work with Aiven for Apache Kafka.
+
+1. Generate a Flink job skeleton named ``capitalizer`` using the Maven archetype:
+
+   .. code:: shell
+
+      mvn archetype:generate -DinteractiveMode=false \
+        -DarchetypeGroupId=org.apache.flink -DarchetypeArtifactId=flink-quickstart-java \
+        -DarchetypeVersion=1.15.1 -DgroupId=com.example -DartifactId=capitalizer \
+        -Dversion=0.0.1-SNAPSHOT
+
+2. Uncomment the Kafka connector in `pom.xml`:
```

Comment:
Should we add `Uncomment` to our list of accepted words? any opinions @TibsAtWork ?

Reply:
I vote yes. Add it (all lowercase) to `.github/vale/dicts/aiven.dic`